docs/articles/faq/README.md

```diff
@@ -259,4 +262,54 @@ and [t.resizeWindow](../documentation/test-api/actions/resize-window.md#setting-
 to control the browser window size. If the test passes, it means your webpage hides
 the target element when the window is resized to smaller dimensions.
 
-Finally, try updating TestCafe to the latest version to see if the problem persists.
\ No newline at end of file
+Finally, try updating TestCafe to the latest version to see if the problem persists.
+
+### TestCafe reports that a request has failed. What are the possible reasons?
+
+When TestCafe does not receive a successful response from a server, it outputs the following error:
+
+```text
+A request to https://www.example.com has failed. Use quarantine mode to perform additional attempts to execute this test.
+```
+
+[Quarantine mode](../documentation/using-testcafe/command-line-interface.md#-q---quarantine-mode) helps you complete the tests if the problem is occasional.
+
+However, we recommend that you determine the cause of this issue and address it.
+
+The following situations lead to this error most frequently:
+
+#### The Web server is not responding
+
+Check if the Web and DNS servers are online and configured to accept requests to this URL.
+
+#### Unstable or improperly configured network connection
+
+To fix network issues, try the following:
```

Comment:
"To fix network issues, try the following:" Is it necessary to add this? The heading is sufficient.